src/parser.coffee

```diff
@@ -301,7 +301,7 @@ class VASTParser
             companionAd.width = companionResource.getAttribute("width")
             companionAd.height = companionResource.getAttribute("height")
             for htmlElement in @childsByName(companionResource, "HTMLResource")
-                companionAd.type = htmlElement.getAttribute("creativeType") or 0
+                companionAd.type = htmlElement.getAttribute("creativeType") or 'text/html'
```

Comment:
I'm not sure about this default and why it shouldn't be reflected for other type of resources (i.e: 3 lines below)?